app/lib/forms/kind_of_nursery.rb

```diff
@@ -1,36 +1,58 @@
 module Forms
   class KindOfNursery < Base
-    attr_accessor :kind_of_nursery
-
     KIND_OF_NURSERY_PUBLIC_OPTIONS = %w[
       local_authority_maintained_nursery
       preschool_class_as_part_of_school
     ].freeze
-    KIND_OF_NURSERY_PRIVATE_OPTIONS = %w[private_nursery].freeze
+
+    KIND_OF_NURSERY_PRIVATE_OPTIONS = %w[
+      private_nursery
+      another_early_years_setting
+    ].freeze
     KIND_OF_NURSERY_OPTIONS = KIND_OF_NURSERY_PUBLIC_OPTIONS + KIND_OF_NURSERY_PRIVATE_OPTIONS
 
+    attr_accessor :kind_of_nursery
+
     validates :kind_of_nursery, presence: true, inclusion: { in: KIND_OF_NURSERY_OPTIONS }
 
     def self.permitted_params
-      %i[
-        kind_of_nursery
-      ]
+      %i[kind_of_nursery]
     end
 
     def next_step
-      if private_nursery?
-        :have_ofsted_urn
-      else
+      if non_ofsted_route?
```

Comment:
Very minor but this makes me think of the rubocop linter that prefers if x over unless !x. Would it be neater to do:
```
if ofsted_route? 
  :have_ofsted_urn
else 
  :find_childcare_provider
end
```

Reply:
yeah possibly declaring the negative harder to read